src/ontology/mondo-edit.obo

```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
+name: IgG4-related systemic disease
+subset: ordo_disease
+synonym: "IgG4-RD" BROAD ABBREVIATION [Orphanet:596448]
+synonym: "IgG4-related disease" BROAD [Orphanet:596448]
+xref: Orphanet:596448 {source="MONDO:equivalentTo"}
+is_a: MONDO:0017287 {source="Orphanet:596448"} ! IgG4-related disease
+
+[Term]
+id: MONDO:0035357
+name: portosinusoidal vascular disease
+subset: ordo_disease
+synonym: "PSVD" EXACT ABBREVIATION [Orphanet:596937]
+xref: ICD10:K76.8 {source="Orphanet:596937/ntbt", source="Orphanet:596937"}
+xref: Orphanet:596937 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002405 {source="Orphanet:596937"} ! hepatic vascular disease
+
+[Term]
+id: MONDO:0035362
+name: TRIM22-related inflammatory bowel disease
+subset: ordo_disease
+synonym: "TRIM22-related IBD" EXACT [Orphanet:597201]
+xref: Orphanet:597201 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005265 {source="Orphanet:597201"} ! inflammatory bowel disease
+
+[Term]
+id: MONDO:0035363
+name: IRF2BPL-related regressive neurodevelopmental disorder-dystonia-seizures syndrome
+subset: ordo_disease
+xref: Orphanet:597623 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/14282 {source="Orphanet:597623"}
+
+[Term]
+id: MONDO:0035369
+name: MTHFS-related developmental delay-microcephaly-short stature-epilepsy syndrome
+subset: ordo_disease
+xref: Orphanet:597874 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000001 {source="Orphanet:597874"} ! disease or disorder
+is_a: MONDO:0017313 {source="Orphanet:597874"} ! disorder of folate metabolism and transport
+is_a: MONDO:0019046 {source="Orphanet:597874"} ! leukodystrophy
+is_a: MONDO:0019058 {source="Orphanet:597874"} ! neurometabolic disease
+
+[Term]
+id: MONDO:0035370
+name: ALPI-related inflammatory bowel disease
+subset: ordo_disease
+xref: Orphanet:597887 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005265 {source="https://orcid.org/0000-0001-5208-3432"} ! inflammatory bowel disease
+
+[Term]
+id: MONDO:0035375
+name: multisystem inflammatory syndrome in children and adults
+subset: ordo_disease
+synonym: "MIS-C/A" EXACT ABBREVIATION [Orphanet:598363]
+xref: ICD10:U10.9 {source="Orphanet:598363/ntbt", source="Orphanet:598363"}
+xref: Orphanet:598363 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015938 {source="Orphanet:598363"} ! systemic disease
+is_a: MONDO:0100320 {source="https://orcid.org/0000-0001-5208-3432"} ! post-COVID-19 disorder
+
+[Term]
+id: MONDO:0035398
+name: hypomyelination of early myelinating structures
+subset: ordo_disease
+synonym: "HEMS" EXACT ABBREVIATION [Orphanet:599376]
+xref: Orphanet:599376 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019046 {source="Orphanet:599376"} ! leukodystrophy
+is_a: MONDO:0100309 {source="Orphanet:599376"} ! hereditary ataxia
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/9086 {source="Orphanet:599376"} ! PLP1
```

Comment:
remember always lookup the gene in omim

https://www.omim.org/entry/300401?search=PLP1&highlight=plp1

is it one of these?

Reply:
I did look at both of those and they don't seem equivalent. I'll double-check this with Ada though.

Reply:
https://github.com/monarch-initiative/mondo/issues/3544